scripts/sysman/powerLimits.yml

```diff
@@ -9,6 +9,8 @@
 type: header
 desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Power Limits"
 version: "1.4"
+details:
+    - "1.7": "Note: This extension is deprecated and merged to $sPower."
```

Comment:
`- "1.7": "[DEPRECATED] Use $sPower."`

Reply:
removing this as we need to define new set of interfaces first before deprecating this